src/scripts/underground/Mine.ts

```diff
@@ -317,7 +363,7 @@ class Mine {
         this.itemsBuried(mine.itemsBuried);
         this.rewardNumbers = mine.rewardNumbers;
         this.loadingNewLayer = false;
-        this.prospectResult(mine.prospectResult ?? this.prospectResult());
+        this.surveyResult(mine.surveyResult ?? this.surveyResult());
```

Comment:
Something will be need to be added to `Update.ts` to ensure this data carries over.

Reply:
I'll look into this.

Reply:
On second thought, do we even want to do this? I quickly tested this and confirmed we lose survey results between versions, but it also allows you to survey again and there's no errors, so I think in all likelihood, players will be more surprised to find 'Prospect' has been reworked to 'Survey' rather than the fact their prospect results are no longer visible. lol 

Also. More pressingly, how would I even go about doing this? 😅